helm/sealed-secrets/Chart.yaml

```
@@ -14,4 +14,4 @@ maintainers:
     url: https://github.com/bitnami-labs/sealed-secrets
 name: sealed-secrets
 type: application
-version: 2.4.0
+version: 2.4.1
```

```suggestion
version: 2.5.0
```
A new minor since you are adding a new parameter.